neo/public/javascripts/map/controllers.js

```diff
@@ -444,10 +464,16 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
             resetCount(geo);
             angular.forEach(result, function (r) {
               if (r[level] === geo['properties'][level+"ID"]){
-                if ($scope.doNormalization)
-                  setNormalizedCount(geo, r);
-                else
-                  setUnnormalizedCount(geo, r);
+                if($scope.doSentiment){
+                  // TODO: change fake random number to real sentiment (0-4)
+                  geo['properties']['count'] = Math.random() * 4;
+                  geo["properties"]["countText"] = geo["properties"]["count"].toFixed(1);
```

Comment:
As often indicated by the codacy, double quote is the norm to follow. There are many places in the project involving double quotes and single quotes. I think we will need a dedicated PR to fix it.